docs/developer-guide.md

```diff
@@ -1,25 +1,56 @@
 # Developer Guidelines
 
-## Continuous Integration (CI)
+## Setup for Local Development
+
+### Prerequisites
+
+1. Install `asdf` if not already installed
+2. Clone this repository
+3. `cd` into the repository
+
+### Build and Run Storybook
+
+- Run `yarn` to install package dependencies (follow any prompts `asdf` provides)
+- Run `yarn llink` to set up symlinks
+- Run `yarn build` to compile all packages
+- Run `yarn storybook:<theme>` where `<theme>` is `bcgov`, `button`, or your custom theme
```

Comment:
These are the instructions Kristen gave me a year ago. We might additionally need `bootstrap`? https://github.com/button-inc/service-development-toolkit/pull/457/commits/fe8808151a04fc1d1352d963818579e0bba73d99

Reply:
yeah running `yarn bootstrap` is necessary before `yarn llink`. I also proposed a similar change to the docs [here](https://github.com/button-inc/service-development-toolkit/pull/478/commits/3d3b9e13a9fc5a9b07c767b4bc63f7b63554a986)